ultrasync/config.py

```diff
@@ -43,9 +43,14 @@ class UltraSyncConfig(object):
 
     _user_agent = 'Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:69.0) ' \
                   'Gecko/20100101 Firefox/69.0'
+    _online = False
+    _master_key = 'QY+6jk22xgoZC5YdKg5qBwOr3kaAbxXgXnKHAfHOvYwBw0iWHtf/MVRG/97Cx2Tt'
```

Comment:
I think there was some discussion in the other chats.... do we not risk 2 things:
1. exposing and leveraging a security vunverability in a specific control panel to which the key could just be replaced on the next firrmware release of it?
2. I would also be concerned if there are legal ramifications to sharing this key for public use here.  Not sure if we can get some official feedback  before i merge?

Thoughts?